app/models/host_aggregate.rb

```diff
@@ -25,6 +25,90 @@ class HostAggregate < ApplicationRecord
 
   virtual_total :total_vms, :vms
 
+  def self.create_aggregate_queue(userid, ext_management_system, options = {})
+    task_opts = {
+      :action => "creating Host Aggregate for user #{userid}",
+      :userid => userid
+    }
+    queue_opts = {
+      :class_name  => ext_management_system.class.class_by_ems("HostAggregate"),
```

Comment:
I did modify this to not hard-code the openstack class, see https://github.com/ManageIQ/manageiq-providers-openstack/pull/778/files#diff-00cff0d589951da9e5c1c3a979716d4e4fe97887c70a1ad414c0466bea9478bfL23 for the old one

Reply:
makes sense now that it's in core.